.gitlab-ci.yml

```diff
@@ -114,13 +114,22 @@ default:
     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
 
 .build-refs:                       &build-refs
```

Comment:
you should create another anchor for these rules. This anchor is used in `build-linux-subkey` and `publish-s3-release` jobs, we do not want to execute them on PRs.

Reply:
should be resolved